### ISSUE TITLE
[bug] Chinese Language

### ISSUE THREAD

**Issue by o-data** [uav4geo/LibreTranslate#19]
Used Online version as test; surprisingly I ran into this bug;

Issue: This phrase is not translated in Chinese

Reproduce: Input English text as per below;

English input: hello how are you

Chinese output:  hello how are you

But Chinese language itself does seem to work though, if you write something else aside from those 4 words for example:

```
English input: hello how are you
how are you
test
test
```

You will get this:

```
英文投入: hello how are you
How are you
测试
测试
```

PS: Thank you for making LibreTranslate. If model training isn't too CPU/GPU intensive (got old PC) and easy to train, I would love to contribute.

**Comment by o-data** [comment]
Might be relevant to this Argous issue (argosopentech/argos-translate#17  I guess this issue can be close.

**Comment by pierotofy** [comment]
Yep, it's due to that issue. We can track it here too, I don't mind.
